src/AppBundle/Resources/views/Report/ProfDeputyCostsEstimate/_answers.html.twig

```diff
@@ -47,89 +47,65 @@
                     </td>
                 {% endif %}
             </tr>
-
-            {# Detailed entries #}
-            {% for estimateCostTypeId in report.getProfDeputyEstimateCostTypeIds %}
-                {% set costRow = submittedEstimateCosts[estimateCostTypeId.typeId] %}
-                <tr class="behat-region-breakdown-{{ estimateCostTypeId.typeId | lower }}">
-                    <td>
-                        {{ ('breakdown.form.entries.' ~ estimateCostTypeId.typeId ~ '.label') | trans }}
-                    </td>
+            {# More information #}
+            {% if report.profDeputyCostsEstimateHasMoreInfo is not empty %}
+                <tr class="behat-region-more-info">
                     <td>
-                        {% if costRow['amount'] is null %}
-                            £{{ 0 | money_format }}
-                        {% else %}
-                            £{{ costRow['amount'] | money_format }}
-                        {% endif %}
+                        {{ 'moreInfo.form.profDeputyCostsEstimateHasMoreInfo.label' | trans }}
                     </td>
-                    {% if not hideEditLink %}
```

Comment:
Can you check this isnt required as i think its pulled in by the review and report PDF templates and has the Edit link in by default ?